demo/src/main/java/com/google/maps/android/utils/demo/ClusteringDemoActivity.java

```diff
@@ -70,6 +70,10 @@ public View getInfoContents(Marker marker) {
                 Toast.makeText(ClusteringDemoActivity.this,
                 "Info window clicked.",
                      Toast.LENGTH_SHORT).show());
+        mClusterManager.getMarkerCollection().setOnInfoWindowLongClickListener(marker ->
```

Comment:
Should this be preferred over setting listeners through `mClusterManager.setOnClusterItemInfoWindowLongClickListener`/`mClusterManager.setOnClusterInfoWindowLongClickListener`?

Reply:
Hmm, good question. To be honest I was following the existing pattern for setOnInfoWindowClickListener() - let me take a closer look.

Reply:
@barbeau , may I apply this change? I think setting the listener directly on the Cluster makes more sense - I can update this file to reflect that.

Reply:
To be honest it's been a while since I've looked at this, and I recall it not being straightforward but I don't recall the details. I'll trust your judgment on what you think is the best path forward. 😁

Reply:
I have added the listener directly to the Cluster. @wangela , if you have some time a review here would be appreciated.